ambry-cloud/src/main/java/com/github/ambry/cloud/azure/AzureStorageCompactor.java

```diff
@@ -264,44 +266,6 @@ private int compactPartitionBucketed(String partitionPath, String fieldName, lon
     }
   }
 
-  /**
-   * Permanently delete the specified blobs in Azure storage.
-   * @param blobMetadataList the list of {@link CloudBlobMetadata} referencing the blobs to purge.
-   * @return the number of blobs successfully purged.
-   * @throws CloudStorageException if the purge operation fails for any blob.
-   */
-  int purgeBlobs(List<CloudBlobMetadata> blobMetadataList) throws CloudStorageException {
```

Comment:
Can we keep this method and make it a wrapper around the utility?  Then callers don't need to know the impl details.